h-youjin/LV.0/양꼬치.js

```diff
@@ -0,0 +1,3 @@
+function solution(n, k) {
+  return n * 12000 + k * 2000 - Math.floor(n / 10) * 2000;
```

Comment:
```suggestion
  return n * 12000 + 2000 * (k - Math.floor(n / 10));
```
뒷부분은 음료 관련 금액으로 2000을 한번에 묶어서 곱해줘도 괜찮을 것 같아요~!!
코드 너무 잘 짜신것 같아요!! 👍 Math.floor() 또 하나의 방법 알아갑니다~!! 🤭